australian-guide-to-legal-citation-3rd-edition.csl

```diff
@@ -0,0 +1,587 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Australian Guide to Legal Citation</title>
```

Comment:
Title should be updated to also say "3rd edition" (without a preceding comma, probably)